shapeshift/web
# feat: add litecoin support

## Description

Adds standard Litecoin support to the platform and makes use of the common `UtxoBaseAdapter` and `UtxoChainId` types to reduce some extra copy/pasta for integrating new utxo coins

## Notice




## Pull Request Type


## Issue (if applicable)



## Risk

Low/Med - with the refactor to the `AppContext.ts` accountSelectors logic and the send logic, there is a slight chance of regression

## Testing

- Ensure all accounts are available and displaying as expected
- Ensure ability to send transactions for available accounts
- Ensure standard litecoin support (send/receive/balance/history) is working

## Screenshots (if applicable)

![image](https://user-images.githubusercontent.com/35275952/181805017-779905f9-ccbe-4cc7-9c0f-c78ec27ea561.png)

![image](https://user-images.githubusercontent.com/35275952/181805054-878fb9ad-e5c7-499e-98f6-c8b22a185dd5.png)

![image](https://user-images.githubusercontent.com/35275952/181805131-4a674957-cf28-41f8-aa91-a7d770d07c56.png)

![image](https://user-images.githubusercontent.com/35275952/181805203-7a5213f5-80bb-476c-b585-5701d25eb29e.png)

![image](https://user-images.githubusercontent.com/35275952/181805313-decffb11-97bc-4ed2-b007-0332161dc46c.png)

![image](https://user-images.githubusercontent.com/35275952/181805361-268dd66b-0c76-41e8-a8ed-eea26cfe5526.png)

![image](https://user-images.githubusercontent.com/35275952/181805455-51b44887-69ff-43e7-920d-144e2f71a16a.png)

![image](https://user-images.githubusercontent.com/35275952/181805659-a5d8a913-fed7-4542-94a4-c245a384e354.png)
